### ISSUE TITLE
[1.16.3] Crash with Charm

### ISSUE THREAD

**Issue by crysthala** [Eilux/BetterSlabs#6]
Crashes before startup with exit code 0. Removing either this mod or [Charm](https://www.curseforge.com/minecraft/mc-mods/charm) allows the game to start. Reporting to the other dev, as well.

[latest.log](https://github.com/Eilux/BetterSlabs/files/5426680/latest.log), no crash-report generated.
**Fabric Loader:** 0.10.1-209
**Fabric API:** 0.2.1-412
**Charm:** 2.0.3
**Better Slabs:** 1.1.1

Please let me know if there's anything I can do to help!

**Comment by svenhjol** [comment]
Conflict with a mixin in Charm.  Fixed in Charm.

I'm using a `@Redirect` because `@Invoke` isn't allowed in the constructor, so I can't reload client-side things here.  I guess you're doing something similar.  Since this isn't the first time that Charm has conflicted with this hook I've disabled it and handled loading textures in a different manner.  Heads up that Better Slabs might end up conflicting with others due to this.

```java
    @Redirect(
        method = "<init>",
        at = @At(
            value = "INVOKE",
            target = "Lnet/minecraft/resource/ResourcePackManager;scanPacks()V"
        )
    )
```
